invenio_rdm_records/ext.py

```diff
@@ -80,6 +84,12 @@ def verify_token():
             session.pop("rdm-records-token", None)
             flash(_("Your shared link has expired."))
 
+    token = request.args.get("access-request-token", None)
```

Comment:
from @anikachurilova: is it safe to store the token's value in the session?

given that it's a server-side session, i think it should be okay; if server-side sessions get leaked, we're probably in more trouble than third parties being able to view access request details.
however, currently i'm letting the system create a comment including the generated secret link on the access request when it is being accepted – maybe we should remove this.

can you think of any potential attack vectors / security issues here?